tutorials/nservicebus-sagas/1-saga-basics/tutorial.md

```diff
@@ -238,7 +238,7 @@ Remember that it's possible that `OrderBilled` may be handled before `OrderPlace
 
 ### Summary
 
-In this lesson, we learned to think of sagas as a tool to implement a business policy. _An order cannot be shipped until it is both **accepted** and **billed**._ We want sagas to react to messages, evaluate business rules, and make decisions that allow the system to move forward. It's generally better to think of sagas as policies rather than as orchestrators or process managers.
+In this lesson, we learned to think of sagas as a tool to implement a business policy. Like _An order cannot be shipped until it is both **accepted** and **billed**._ We want sagas to react to messages, evaluate business rules, and make decisions that allow the system to move forward. It's generally better to think of sagas as policies rather than as orchestrators or process managers.
```

Comment:
The last sentence still feels awkward, but not sure if this is any better. I think it should be changed though because we specifically refer to sagas as orchestrators blah blah elsewhere.

```suggestion
In this lesson, we learned to think of sagas as a tool to implement business policies, like _An order cannot be shipped until it is both **accepted** and **billed**._ We want sagas to react to messages, evaluate business rules, and make decisions that allow the system to move forward. It's helpful to think of sagas as policies rather than as orchestrators or process managers.
```